.github/workflows/release.yaml

```diff
@@ -72,3 +73,40 @@ jobs:
             Freeciv21-${{github.ref_name}}.tar.gz.sha256
             Freeciv21-${{github.ref_name}}.zip
             Freeciv21-${{github.ref_name}}.zip.sha256
+
+  snapcraft:
+    name: "Snap Package"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release'
```

Comment:
Maybe we could avoid duplicating the code by moving this `if` to the upload steps?